packages/paste-theme/src/utils/isDeprecatedTheme.ts

```diff
@@ -0,0 +1,11 @@
+import type {DeprecatedThemeVariants} from '../constants';
+
+export const isDeprecatedTheme = (theme: DeprecatedThemeVariants): null => {
+  const themeToDeprecationMessage: Record<DeprecatedThemeVariants, string> = {
```

Comment:
Nice!

Reply:
This is a cool way of writing the error message